src/screens/Wallet.js

```diff
@@ -459,8 +460,12 @@ class Wallet extends React.Component {
 
     const renderUnlockedWallet = () => {
       let template;
-      if (tokenHistory.status === TOKEN_DOWNLOAD_STATUS.LOADING
-          || tokenBalance.status === TOKEN_DOWNLOAD_STATUS.LOADING) {
+      /**
+       * We only show the loading message if we are syncing the entire history
+       * This will happen on the first history load and if we lose connection
+       * to the fullnode.
+       */
+      if (this.props.walletState === hathorLib.HathorWallet.SYNCING) {
```

Comment:
What are we doing if a new tx arrives in the wallet? I thought we were loading the history from the wallet lib. In that case, we should show the loading message as well, shouldn't we?

Reply:
The component used to show the transactions as a table will show a loading message when we send the event to load the history.